src/hooks/useTokensMarketPrice.ts

```diff
@@ -44,7 +44,9 @@ export default function useTokensMarketPrice(tokens: (Token | null | undefined)[
 
   const tokenAddress = tokens
     .filter(Boolean)
-    .map(token => (token?.isNative ? WETH[chainId || ChainId.MAINNET].address : token?.address))
+    .map(token =>
+      token?.isNative || token?.address === ZERO_ADDRESS ? WETH[chainId || ChainId.MAINNET].address : token?.address,
```

Comment:
Why do we need both `token?.isNative` and `token?.address === ZERO_ADDRESS`. Doesnt they are same and always go together?

Reply:
native token doesnt have address

Reply:
zero address is only a trick for reward token of farm

Reply:
These trick logic make it alot harder to maintain, should reduce them as much as possible in the future.